client/header/activity-panel/panels/stock/index.js

```diff
@@ -0,0 +1,99 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component, Fragment } from '@wordpress/element';
+import { compose } from '@wordpress/compose';
+import PropTypes from 'prop-types';
+
+/**
+ * WooCommerce dependencies
+ */
+import { EmptyContent, Section } from '@woocommerce/components';
+
+/**
+ * Internal dependencies
+ */
+import { ActivityCardPlaceholder } from '../../activity-card';
+import ActivityHeader from '../../activity-header';
+import ProductStockCard from './card';
+import { QUERY_DEFAULTS } from 'wc-api/constants';
+import withSelect from 'wc-api/with-select';
+
+class StockPanel extends Component {
+	render() {
+		const { isError, isRequesting, products } = this.props;
+
+		if ( isError ) {
+			const title = __(
+				'There was an error getting your low stock products. Please try again.',
+				'woocommerce-admin'
+			);
+			const actionLabel = __( 'Reload', 'woocommerce-admin' );
+			const actionCallback = () => {
+				// @todo Add tracking for how often an error is displayed, and the reload action is clicked.
+				window.location.reload();
```

Comment:
This is a separate issue here, but doesn't this lend itself to a potential endless loop? An app-wide `reload` function which inserts a query parameter to signal an attempt to reload has already been made could be useful.

Reply:
Are you considering the page reloading to another error state an endless loop, or something else?

This is the same logic for the Inbox and Orders panels - an issue created to resolve should fix those as well.